src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -113,6 +114,7 @@ public class ServerStore : IDisposable
 
         private readonly NotificationsStorage _notificationsStorage;
         private readonly OperationsStorage _operationsStorage;
+        private readonly LuceneCleaner _luceneCleaner;
```

Comment:
static?

Reply:
also move to LuceneIndexReadOperation maybe?

Reply:
moved. will need to adjust in v5.4